src/core/proof.ml

```diff
@@ -4,103 +4,209 @@ open! Lplib
 open Lplib.Base
 
 open Timed
-open Pos
 open Terms
 open Print
 
 (** Abstract representation of a goal. *)
 module Goal :
   sig
```

Comment:
Is the sig really necessary here? Do you need to hide some elements?

Reply:
I don't know why there was a signature here... As we don't use mli files, everything (but overwritten ocaml symbols) is public. I suppose it's more coherent to remove this signature... I don't have any particular opinion on this. I removed it...